fatal error C1083: 无法打开包括文件: “mysql.h”: No such file or directory

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
{
  "dependencies": [
    {
      "name": "curl",
      "features": [
        "ssh"
      ]
    },
    "fmt",
    "libssh2",
    "redis-plus-plus",
    {
      "name": "sqlpp11",
      "features": [
        "mysql",
        "sqlite3"
      ]
    },
    "mysql-connector-cpp"
  ]
}
```

### Failure logs

[build] E:\develop\candcpp\cmakedemo\out\build\vs2022_x64\vcpkg_installed\x64-windows-static-md\include\sqlpp11/mysql/sqlpp_mysql.h(29,10): fatal error C1083: 无法打开包括文件: “mysql.h”: No such file or directory

### Additional context

Host Environment

Host: x64-windows
Compiler: MSVC 19.31.31106.2
vcpkg-tool version: 2024-11-12-eb492805e92a2c14a230f5c3deb3e89f6771c321
vcpkg-scripts version: https://github.com/microsoft/vcpkg/commit/b2cb0da531c2f1f740045bfe7c4dac59f0b2b69c 2024-11-15 (9 weeks ago)

I change this:#include <mysql/mysql.h>,cmake successed!
but build error
:[build] E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\vcpkg_installed\x64-windows-static-md\include\sqlpp11/mysql/bind_result.h(325,51): error C2589: “(”:“::”右边的非法标记 [E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\cvv\src\ssqlpp11\test\ssqlpp11_test.vcxproj]
[build] E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\vcpkg_installed\x64-windows-static-md\include\sqlpp11/mysql/bind_result.h(325): error C2062: 意外的类型“unknown-type” [E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\cvv\src\ssqlpp11\test\ssqlpp11_test.vcxproj]
[build] E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\vcpkg_installed\x64-windows-static-md\include\sqlpp11/mysql/bind_result.h(325,1): error C2059: 语法错误:“)” [E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\cvv\src\ssqlpp11\test\ssqlpp11_test.vcxproj]
[build] E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\vcpkg_installed\x64-windows-static-md\include\sqlpp11/mysql/bind_result.h(326,1): error C2143: 语法错误: 缺少“;”(在“{”的前面) [E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\cvv\src\ssqlpp11\test\ssqlpp11_test.vcxproj]
[build] E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\vcpkg_installed\x64-windows-static-md\include\sqlpp11/mysql/bind_result.h(326,90): error C2143: 语法错误: 缺少“;”(在“}”的前面) [E:\develop\candcpp\cmakedemo\out\build\cmakemsvcvcpkg\cvv\src\ssqlpp11\test\ssqlpp11_test.vcxproj]
[proc] 命令“"C:\Program Files\CMake\bin\cmake.EXE" --build E:/develop/candcpp/cmakedemo/out/build/cmakemsvcvcpkg --config Debug --”已退出，代码为 1
[driver] 生成完毕: 00:00:07.613

--- a/ports/sqlpp11/fix-miss-header.patch
+++ b/ports/sqlpp11/fix-miss-header.patch
@@ -1,0 +1,13 @@
+diff --git a/include/sqlpp11/mysql/sqlpp_mysql.h b/include/sqlpp11/mysql/sqlpp_mysql.h
+index 4257c7a..1408037 100644
+--- a/include/sqlpp11/mysql/sqlpp_mysql.h
++++ b/include/sqlpp11/mysql/sqlpp_mysql.h
+@@ -26,7 +26,7 @@
+  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+  */
+ 
+-#include <mysql.h>
++#include <mysql/mysql.h>
+ 
+ namespace sqlpp
+ {

--- a/ports/sqlpp11/portfile.cmake
+++ b/ports/sqlpp11/portfile.cmake
@@ -9,6 +9,7 @@ vcpkg_from_github(
     PATCHES
         ddl2cpp_path.patch
         dependencies.diff
+        fix-miss-header.patch
 )
 
 vcpkg_check_features(

--- a/ports/sqlpp11/vcpkg.json
+++ b/ports/sqlpp11/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "sqlpp11",
   "version": "0.64",
-  "port-version": 2,
+  "port-version": 3,
   "description": "A type safe embedded domain specific language for SQL queries and results in C++.",
   "homepage": "https://github.com/rbock/sqlpp11",
   "license": "BSD-2-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8694,7 +8694,7 @@
     },
     "sqlpp11": {
       "baseline": "0.64",
-      "port-version": 2
+      "port-version": 3
     },
     "sqlpp11-connector-mysql": {
       "baseline": "0.61",

--- a/versions/s-/sqlpp11.json
+++ b/versions/s-/sqlpp11.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "dfc7252b01f1c44f133b24c47a245bed2173b317",
+      "version": "0.64",
+      "port-version": 3
+    },
+    {
       "git-tree": "16c9c831d703248a11117054ee7974a6381fe411",
       "version": "0.64",
       "port-version": 2

